### ISSUE TITLE
All interface text disappearing, although inconsistently

### ISSUE THREAD

**Issue by the-og-gear** [al132mc/antimatter-chemistry#262]
As the title says, all interface text is disappearing. Item stack counts, F3 debug menus, item names, legitimately everything that is displayed with plaintext is just... gone. There are no log outputs when this happens, no game lag, text just disappears.

![image](https://user-images.githubusercontent.com/9438930/98391959-5660df00-2025-11eb-9d67-85138d47e97b.png)

This also happens in menus, including out on the main menu. It takes a full client restart to fix, but honestly this isn't feasible to do when it tends to happen upon opening an inventory and is so wildly inconcsistant that it could be within the first minute of loading up a world or an hour in to playing.

![image](https://user-images.githubusercontent.com/9438930/98392136-958f3000-2025-11eb-96c2-8ea4187295ca.png)

![image](https://user-images.githubusercontent.com/9438930/98392180-a3dd4c00-2025-11eb-9d8e-edaa1edbe8ef.png)

I had installed the Unity Dark resource pack in an attempt to fix by changing the font, however it doesn't seem to change the font anyways. This was happening without resource packs on a fresh instance install. I've also attempted a texture reload from within the game with F3+T, and exiting to the main menu and reloading the world. Nothing I've tried so far short of restarting the entire client reloads the text.

**Comment by ghost** [comment]
Honestly I have no idea... someone else please comment if you do.

**Comment by the-og-gear** [comment]
I've been able to delay how long it takes to happen by removing FoamFix and disabling the Forge splash screen in splash.properties, and at this point it's deciding to remove all text upon searching JEI within AE2 terminals, or just seemingly randomly within inventories. It is nowhere near consistent with the randomness and has happened exactly once since, but the JEI search happens every single time. It's... Quite odd, honestly.

**Comment by the-og-gear** [comment]
So it seems while I thought I had the correct drivers, Windows decided that version 29 of Intel's graphics drivers were correct when really only 27 supports my processor. For *some reason*, this didn't cause issue with other packs or games, and it was only this one. I've been going strong for about four hours now without a single issue since fixing this, although I won't hit close right away in case this wasn't the solution.

In short, **check your drivers manually** instead of letting Windows handle them. Yeah.
